src/pages/home/ReportScreen.js

```diff
@@ -89,6 +90,9 @@ class ReportScreen extends React.Component {
      */
     storeCurrentlyViewedReport() {
         const reportID = this.getReportID();
+        if (_.isNaN(reportID)) {
+            handleInaccessibleReport();
```

Comment:
Can you also add a return here so that we don't call `updateCurrentlyViewedReportID` with `NaN`?